cubano-config/src/main/java/org/concordion/cubano/utils/CaselessProperties.java

```diff
@@ -1,30 +1,40 @@
-package org.concordion.cubano.utils;
-
-import java.util.Properties;
-
-/**
- * A {@link Properties} implementation that ignores the case of the properties.
- *
- * @author Andrew Sumner
- */
-public class CaselessProperties extends Properties {
-    private static final long serialVersionUID = 4112578634029874840L;
-
-    @Override
-    public Object put(Object key, Object value) {
-        String lowercase = ((String) key).toLowerCase();
-        return super.put(lowercase, value);
-    }
-
-    @Override
-    public String getProperty(String key) {
-        String lowercase = key.toLowerCase();
-        return super.getProperty(lowercase);
-    }
-
-    @Override
-    public String getProperty(String key, String defaultValue) {
-        String lowercase = key.toLowerCase();
-        return super.getProperty(lowercase, defaultValue);
-    }
-}
+package org.concordion.cubano.utils;

```

Comment:
Let's also add a setting to keep line endings consistent whether editing on Mac or Windows. See https://help.github.com/articles/dealing-with-line-endings/.

Reply:
Added .gitattributes file as per the article